src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
This seems like the easiest, most cross-platform way to accomplish a true "return ignore":

```
template<typename T>
inline void ignore_result(T /* unused result */) {}
```

Allegedly (void) casting is going out of vogue...